<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffsForm</class>
 <widget class="QWidget" name="StaffsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StaffsForm</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="editBtn">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="deleteBtn">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="closeBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="1" column="1">
     <widget class="QListWidget" name="designersListWidget"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="managersLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Менеджеры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="designersLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дизайнеры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addManagerBtn">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="addDesignerBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="managersListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>managersListWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>StaffsForm</receiver>
   <slot>slot5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>designersListWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>StaffsForm</receiver>
   <slot>slot6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addManagerBtn</sender>
   <signal>clicked()</signal>
   <receiver>StaffsForm</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDesignerBtn</sender>
   <signal>clicked()</signal>
   <receiver>StaffsForm</receiver>
   <slot>slot4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editBtn</sender>
   <signal>clicked()</signal>
   <receiver>StaffsForm</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBtn</sender>
   <signal>clicked()</signal>
   <receiver>StaffsForm</receiver>
   <slot>slot3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>StaffsForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
  <slot>slot4()</slot>
  <slot>slot5()</slot>
  <slot>slot6()</slot>
 </slots>
</ui>
